avs-aggregator/bindings/FinalizerServiceManager/binding.go

```diff
@@ -38,8 +38,8 @@ type ISignatureUtilsSignatureWithSaltAndExpiry struct {
 
 // ContractFinalizerServiceManagerMetaData contains all meta data concerning the ContractFinalizerServiceManager contract.
 var ContractFinalizerServiceManagerMetaData = &bind.MetaData{
```

Comment:
I think we missed in the knowledge-share meeting how to update those bindings, :thinking:

Reply:
yes
there are commands in the Makefile, to generate both golang and rust bindings